sdks/python/apache_beam/runners/interactive/background_caching_job.py

```diff
@@ -259,21 +269,31 @@ def is_source_to_cache_changed(
   # The computation of extract_unbounded_source_signature is expensive, track on
   # change by default.
   if is_changed and update_cached_source_signature:
-    if ie.current_env().options.enable_capture_replay:
-      # TODO(BEAM-8335): display rather than logging when is_in_notebook.
+    options = ie.current_env().options
+    # No info needed when capture replay is disabled.
+    if options.enable_capture_replay:
+
+      def sizeof_fmt(num, suffix='B'):
```

Comment:
Should this be moved inside the if below? Is it used anywhere else?

Reply:
No, it's not used anywhere else. Moving it.